iOS/DuckDuckGo/AppConfiguration/ATBAndVariantConfiguration.swift

```diff
@@ -23,7 +23,16 @@ import BrowserServicesKit
 
 final class ATBAndVariantConfiguration {
 
-    lazy var variantManager = DefaultVariantManager()
+    private let returningUserMeasurement = KeychainReturnUserMeasurement()
+
+    lazy var variantManager: DefaultVariantManager = {
+        DefaultVariantManager(returningUserMeasurement: returningUserMeasurement)
+    }()
+
+    init() {
+        // Deliberately trigger this early so that `isReturningUser` can be cached.
+        _ = returningUserMeasurement.isReturningUser
```

Comment:
This is the key change - we now cache this as soon as `ATBAndVariantConfiguration` is set up. I had to make some of the Core logic public to do this, please suggest a better approach if you see it. Thanks!